src/pybamm/models/base_model.py

```diff
@@ -426,6 +429,11 @@ def input_parameters(self):
             self._input_parameters = self._find_symbols(pybamm.InputParameter)
         return self._input_parameters
 
+    @property
+    def calc_esoh(self):
+        "Whether to include eSOH variables in the summary variables."
```

Comment:
Minor but doc strings should be triple quotes
```suggestion
        """Whether to include eSOH variables in the summary variables."""
```